geode-core/src/upgradeTest/java/org/apache/geode/internal/cache/rollingupgrade/RollingUpgradePartitionRegionClearServerVersionMismatch.java

```diff
@@ -109,20 +111,28 @@ public void before() {
       region.put("A", "ValueA");
       region.put("B", "ValueB");
     });
+  }
+
+  @After
+  public void after() {
+    locator.stop();
+    serverNew.stop();
+    serverOld.stop();
 
   }
 
```

Comment:
Change all the ServerOperationException  to be UnsupportedOperationException in this class, including comments and method names.